### PR TITLE
SOC-1279 Expose servicesDomain to ember and use for discussion api url

### PR DESCRIPTION
**Pull request by armonr**
/CC @kenkouot @mklucsarits @garthwebb 

https://wikia-inc.atlassian.net/browse/SOC-1279

### CODE REVIEW

File: config/localSettings.base.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ var deepExtend: any = require('deep-extend');
 
 var localSettings: LocalSettings = {
 	apiBase: '/api/v1',
-	servicesDomain: 'services.wikia.com',
+	servicesDomain: 'services.wikia-dev.com',
```
Comment (mklucsarits) [inline config/localSettings.base.ts]: The dev URLs are already configured in localSettings.dev.ts. This one in base shouldn't be changed.

**Comment by mklucsarits** [comment]
LGTM

File: front/scripts/main/models/DiscussionIndexModel.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ App.DiscussionIndexModel.reopenClass({
 	find(wikiId: number) {
 		return new Em.RSVP.Promise((resolve: Function, reject: Function) => {
 			Em.$.ajax({
-				url: `https://services.wikia.com/discussion/${wikiId}/forums`,
+				url: 'https://' + M.prop('servicesDomain') + `/discussion/${wikiId}/forums`,
```
Comment (lizlux) [inline front/scripts/main/models/DiscussionIndexModel.ts]: Should the protocol be included in the `servicesDomain`? Would be cleaner from the perspective of someone using the var. Is there any use case for `http` here?

Comment (garthwebb) [inline front/scripts/main/models/DiscussionIndexModel.ts]: The `services.wikia.com` domain redirects to `HTTPS` if you try to access it via `HTTP`.  I'd say leave it for now and if it becomes necessary in the future for some reason we add `M.prop('servicesProto')`

**Comment by garthwebb** [comment]
LGTM!

**Pull request merged**